clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -2445,6 +2445,46 @@ static bool isTypeSubstitutable(Qualifiers Quals, const Type *Ty,
   return true;
 }
 
+namespace {
```

Comment:
Anonymous namespace seems to just accomplish what making the function static does...

Reply:
Moved function out of it, but left it to guard `DeclContextDesc` definition